### DISCUSSION TITLE
How to reformat a map created from a branch?

### DISCUSSION THREAD

**Discussion by halfbeing** [freeplane/freeplane#736] [Help]
I created a new map using "move branch to new map". Unfortunately, the new map is very cramped and lopsided, and looks nothing like the format of the map it was created from. I have tried to restore that formatting by using the "Manage styles" options in the Format menu, but it makes no difference.

What I really want is to be able to completely unlink the new map from its parent, and thus to be able to get the central node exactly how I want it, without the hyperlink back to the parent map. And, of course, I also want the map to have the same formatting as its parent, with branches that are not too short, and that are not all on one side of the central node.

How can I achieve this?

**Comment by EdoFro** [comment]
when exporting a branch to a new map, it has the same styles and formating as its source map, but there is no other mutual relationship between both, just the links between them (that can be easily deleted and have no further implications than opening the other map if clicked)

to change a branch to the other side of the root node you have two options:

1. leftclick the first node of the branch and drag and drop it on the root node. Drop it when the mouse is over the the wanted side of the node
2. select the node you want to move. Use Ctrl + arrow keys to move it to its new position.

**Reply by halfbeing** [reply]
Thanks for the reply. 

I can't see how to remove the link from the root node of the new map. It's not at all obvious.

**Reply by halfbeing** [reply]
Now I see.

"Remove link" is accessible via the "Insert" menu. This is a design flaw. On the insert menu I would expect to find only insert operations.

**Reply by dpolivaev** [reply]
Instead of writing "this is a design flow" I wish you had written, "it has surprised me".
Generally, placing "remove" operations in "insert" menus is a convention some people find reasonable.

Regards.

**Comment by euu2021** [comment]
>And, of course, I also want the map to have the same formatting as its parent, with branches that are not too short [...]

Do you mean, horizontally short? Did you position the branches manually on the original map?

**Comment by halfbeing** [comment]
Sorry, I didn't mean to offend.

I get that some people will look for remove operations in an insert menu,
and if I had created the link myself using the insert menu, I too would
have naturally gone to the same menu to remove it, but it is not inherently
obvious. While I understand that you don't want contextual menus to be
overloaded, in my opinion there is a case for putting link creation and
removal options in the relevant contextual menu, in addition to their place
in the Insert menu.

As for the shortness of the branches, that was just a misunderstanding of
the style template on my part. The node I opened in a new map had no
grandchildren, which was the only reason that the branches were much
shorter than on the original map. As I added sub-nodes to the main topic
nodes, the style of the map started to look more familiar.
